dev/ci/ci-mtac2.sh

```diff
@@ -4,7 +4,7 @@ ci_dir="$(dirname "$0")"
 . "${ci_dir}/ci-common.sh"
 
 unicoq_CI_DIR=${CI_BUILD_DIR}/unicoq
-metacoq_CI_DIR=${CI_BUILD_DIR}/MetaCoq
+mtac2_CI_DIR=${CI_BUILD_DIR}/Mtac2
```

Comment:
This script could be split in 2 (ie adding a ci-unicoq script). You would have to add the dependency information in Makefile.ci and add a unicoq job and dependency info in the circle CI yml (look at the `requires:` entries).